tests/travis/functional_ee.sh

```diff
@@ -8,12 +8,15 @@ trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit
 
 case $TRAVIS_PHP_VERSION in
```

Comment:
Looks like you don't need case here, as in all variants logic the same:
`./vendor/bin/codecept run -g parallel_71_"$FUNCTIONAL_INDEX" --steps`
Just change too 
`./vendor/bin/codecept run -g parallel_"$TRAVIS_PHP_VERSION"_"$FUNCTIONAL_INDEX" --steps` instead of all case statement.
Only need to remove dot `.` in `$TRAVIS_PHP_VERSION`

Reply:
Thanks.
I fixed it.